demisto_sdk/commands/common/tools.py

```diff
@@ -4454,3 +4454,20 @@ def remove_nulls_from_dictionary(data):
     for key in list_of_keys:
         if data[key] in ("", None, [], {}, ()):
             del data[key]
+
+
+def safe_get_relative_path(file_path: Union[str, Path], relative_to: Path) -> Path:
+    """Extract the relative path that is relative to the given path.
+
+    Args:
+        file_path (Union[str, Path]): The path to extract the relative path from.
+        relative_to (Path): The path to get the relative path to.
+
+    Returns:
+        Path: The extracted relative path.
+    """
+    if isinstance(file_path, str):
+        file_path = Path(file_path)
+    if file_path.is_absolute():
+        file_path = file_path.relative_to(relative_to)
```

Comment:
this can raise exception if the relative_to isn't a real relative path

Reply:
In that case I think it's okay for it to fail, or do we want to return the abs path in that case?

Reply:
let it fail, but change the func name to `get_relative_path` because safe gives it a context like it won't raise any exception